commit 1ac8fad8e56097cf8e08a0a17f044309317cb50f
Author: Kay Sievers <kay.sievers@vrfy.org>
Date:   Wed Dec 3 01:32:00 2008 +0100

    rules: fix isdn rules
    
    On Tue, Dec 2, 2008 at 21:07, Matthias Schwarzott <zzam@gentoo.org> wrote:
    > It seems that the rules related to capi devices are not correct:
    >
    > KERNEL=="capi",         NAME="capi20", SYMLINK+="isdn/capi20"
    > KERNEL=="capi*",        NAME="capi/%n"
    >
    > Changing the second rule to match only on KERNEL=="capi[0-9]*" is reported to
    > make it work.
    > So I can only guess that the problem is the second rule overwriting the NAME
    > set by the first one.

diff --git a/rules/packages/40-isdn.rules b/rules/packages/40-isdn.rules
index f2bc34e..8a4686f 100644
--- a/rules/packages/40-isdn.rules
+++ b/rules/packages/40-isdn.rules
@@ -1,5 +1,4 @@
 # do not edit this file, it will be overwritten on update
 
-SUBSYSTEM=="capi",	GROUP="uucp"
-KERNEL=="capi",		NAME="capi20", SYMLINK+="isdn/capi20"
-KERNEL=="capi*",	NAME="capi/%n"
+SUBSYSTEM=="capi", KERNEL=="capi", NAME="capi20", SYMLINK+="isdn/capi20", GROUP="uucp"
+SUBSYSTEM=="tty", KERNEL=="capi[0-9]*", NAME="capi/%n"
